### PR TITLE
OpenAI v1 embeddings errors

### PR DESCRIPTION
**Pull request by umaannamalai**
# Overview
Add support for openai v1 embedding error traces and events.

### CODE REVIEW

File: tests/mlmodel_openai/test_embeddings_error_v1.py [hunk 1]
```diff
@@ -12,17 +12,305 @@
...
+    exact_attrs={
+        "agent": {},
+        "intrinsic": {},
+        "user": {},
+    },
+)
+@validate_span_events(
+    exact_agents={
+        "error.message": "create() missing 1 required keyword-only argument: 'model'"
+        if sys.version_info < (3, 10)
```
Comment (hmstepanek) [inline tests/mlmodel_openai/test_embeddings_error_v1.py:72]: Ugh that's annoying.

File: tests/mlmodel_openai/test_embeddings_error_v1.py [hunk 2]
```diff
@@ -12,17 +12,305 @@
...
+
+
+@dt_enabled
+@reset_core_stats_engine()
+@validate_error_trace_attributes(
+    callable_name(openai.InternalServerError),
+    exact_attrs={
+        "agent": {},
+        "intrinsic": {},
+        "user": {},
```
Comment (hmstepanek) [inline tests/mlmodel_openai/test_embeddings_error_v1.py:160]: I'd expect a minimum to have a status code in here but also usually the error.code is populated as well. I'm guessing these were removed cause the api was miss-behaving yesterday but since we are mocking these should be consistent.
```suggestion
        "user": {
            'http.statusCode': 404,
            'error.code': 'model_not_found',
        },
```

File: tox.ini [hunk 3]
```diff
@@ -495,4 +495,4 @@ source =
 directory = ${TOX_ENV_DIR-.}/htmlcov
 
 [coverage:xml]
-output = ${TOX_ENV_DIR-.}/coverage.xml
+output = ${TOX_ENV_DIR-.}/coverage.xml
```
Comment (hmstepanek) [inline tox.ini:498]: This looks like a no-op change-just a line ending so just remove this tox.ini from the changes.

File: tests/mlmodel_openai/conftest.py [hunk 4]
```diff
@@ -65,7 +65,6 @@
         "test_chat_completion.py",
         "test_get_llm_message_ids.py",
         "test_chat_completion_error.py",
```
Comment (hmstepanek) [inline tests/mlmodel_openai/conftest.py:67]: I made a commit to remove this line. I had added it for my PR since these tests were failing in my branch. I think because the tests weren't actually running before and now they are there's some errors that need to be fixed in them.

**Review by hmstepanek — CHANGES_REQUESTED** [review]
Mainly just need to fix up the tests now that they are actually running and there's a couple minor fixups in the hook file to handle None cases.

**Review by hmstepanek — APPROVED** [review]
Looks good!
